src/test/resources/application-test.properties

```diff
@@ -1,4 +1,4 @@
-spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
why did you change this?

Reply:
I was just trying different things, would switch back